### DISCUSSION TITLE
Session management & multi user

### DISCUSSION THREAD

**Discussion by ChrisLinnNuvisan** [ParisNeo/Gpt4All-webui#28] [Ideas]
Hi,

are there any plans on introducing session management, to enable multi-user environments? Having one webserver with enough resources could be nice. Users should of course not be able to see the discussions other users had. 

Best regards,
Chris

**Comment by ParisNeo** [comment]
Hi. I am thinking of a simpler way to make this happen.
In fact, the lollms backend has a webserver that can run générations on a server and doesn't store the output. So, maybe, make the front-end handle the multi-user stuff, store data on your local database and use your local personal folder for private stuff.
This can be done right now by any third party who wants to build a front end.

The webui i built is made for personal use. The initial promiss of this whole project is keeping your data on your pc. You can run this in a jungle with no connection and ask for survival tips if you want. Nothing gets out of your pc.

**Comment by ParisNeo** [comment]
And by the way. Gpt4all webui is deprecated. I'm keeping it here just for history. The new project is lollms and lollms-webui.

Lord of large language model is a concept to cover all that is happening in the field of llms. One tool to rule them all.
